descriptor/qsarmolecular/src/main/java/org/openscience/cdk/geometry/surface/Point_Type.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2004-2007  The Chemistry Development Kit (CDK) project
```

Comment:
It this file a copy of an existing CDK file? If not, please list here 2019 and your own name or that of you employer.